src/aligned_memory.rs

```diff
@@ -5,31 +5,31 @@ use std::mem;
 
 /// Provides u8 slices at a specified alignment
 #[derive(Clone, Debug, PartialEq)]
-pub struct AlignedMemory {
+pub struct AlignedMemory<const N: usize> {
```

Comment:
This aligned mem change looks like cleanup, could this come in on a separate PR?

Reply:
I've split the AlignedMemory fixes in https://github.com/solana-labs/rbpf/pull/351